ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1094 @@
+  The `custom-config.xml` descriptor makes possible to authenticate the transaction recovery after a failure.
+
+[source,sh,subs="+quotes,attributes+"]
+----
+# go to the directory with distribution of server1
+cd ${jbossHomeName}_1
+./bin/jboss-cli.sh \
+  --file=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/remote-configuration.cli \
+  --properties=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/cli.local.properties
+----
+
+NOTE: For Windows, use the `bin\jboss-cli.bat` script.
+
+[#_configure_datasources]
+=== Configure datasources
+
+The EJBs in this quickstart perform transactional work against a database so the servers need
+to know how to connect to that database. The following two options show how to configure
+an XA datasource with the name `ejbJtaDs` for connecting to either an embedded H2 database
+or to a PostgreSQL database
+
+==== Option 1: Running with embed H2 database
+
+Use of the embedded H2 database is easy, well suited for local testing showcase.
+It's not proper, in any manner, for production use,
+and it's not suited for the Kubernetes/OpenShift environment either.
+
+You may set up the H2 XA datasource with the following CLI script for each {productName} instance.
+
+First setup the first server `server1`.
+
+[source,sh,subs="+quotes,attributes+"]
+----
+cd ${jbossHomeName}_1
+./bin/jboss-cli.sh \
+  --file=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/h2-embedded-configuration.cli \
+  --properties=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/cli.local.properties
+----
+
+Then setup the `server2` and `server3`.
+
+[source,sh,subs="+quotes,attributes+"]
+----
+cd ${jbossHomeName}_2
+./bin/jboss-cli.sh \
+  --file=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/server/extensions/h2-embedded-configuration.cli \
+  --properties=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/server/extensions/cli.local.properties
+cd ${jbossHomeName}_3
+./bin/jboss-cli.sh \
+  --file=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/server/extensions/h2-embedded-configuration.cli \
+  --properties=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/server/extensions/cli.local.properties
+----
+
+NOTE: For Windows, use the `bin\jboss-cli.bat` script.
+
+==== Option 2: Running with PostgreSQL database
+
+First you need a database running. For local testing purposes
+could be used simple docker one-liner like:
+
+[source,sh]
+----
+docker run -p 5432:5432 --rm  -ePOSTGRES_DB=test -ePOSTGRES_USER=test -ePOSTGRES_PASSWORD=test postgres:9.4 -c max-prepared-transactions=110 -c log-statement=all
+----
+
+The following is a brief summary of the required steps for configuring PostgreSQL,
+refer to https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/CONFIGURE_POSTGRESQL.md#download-and-install-postgresql[CONFIGURE_POSTGRESQL.md]
+for more complete details.
+
+Here we present only a quick summary of the steps:
```

Comment:
Delete this line.  You said the same thing above.

Reply:
+1